src/monomial.jl

```diff
@@ -73,7 +73,7 @@ degree(v::AbstractVariable, var::AbstractVariable) = (v == var ? 1 : 0)
 #degree(m::AbstractMonomial, v::AbstractVariable) = _deg(v, powers(t)...)
 function degree(m::AbstractMonomial, v::AbstractVariable)
     i = findfirst(variables(m), v)
-    if iszero(i)
+    if i == nothing || iszero(i)
```

Comment:
I think it is better to do `=== nothing`

Reply:
Just checked
```julia
@edit ==(nothing, nothing)
```
shows that the definition used is simply
```julia
==(x, y) = x === y
```

Since I assume the compiler will inline this, it should be fine :)

Reply:
(I merged the maybe a little bit too fast, sorry for that)

Reply:
No worry, it wasn't too fast ;) it is a personal preference to use `===` when comparing to nothing, I find it more readable but no big deal. However I didn't know about the `@edit` macro, it will save me so much time, thank you ! I also discovered `@which` recently which is similar.